src/EditorFeatures/VisualBasicTest/AddParameter/AddParameterTests.vb

```diff
@@ -417,5 +423,598 @@ class C
+
+        <WorkItem(21446, "https://github.com/dotnet/roslyn/issues/21446")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)>
+        Public Async Function TestInvocationTuple1() As Task
+            ' error BC30057: Too many arguments to 'Private Sub M1(t1 As (Integer, Integer))'.
+            Dim code =
+"
+Friend Class C1
+    Private Sub M1(t1 As (Integer, Integer))
+        Me.M1((1,1), [|(1,""1"")|])
+    End Sub
+End Class"
+            Dim fix =
+"
+Friend Class C1
+    Private Sub M1(t1 As (Integer, Integer), p As (Integer, String))
+        Me.M1((1,1), (1,""1""))
+    End Sub
+End Class"
+            Await TestInRegularAndScriptAsync(code, fix)
+        End Function
+
+        <WorkItem(21446, "https://github.com/dotnet/roslyn/issues/21446")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)>
+        Public Async Function TestInvocationTuple2() As Task
+            ' error BC30057: Too many arguments to 'Private Sub M1(t1 As (Integer, Integer))'.
+            Dim code =
+"
+Friend Class C1
+    Private Sub M1(t1 As (Integer, Integer))
+        Dim tup=(1, ""1"")
+        Me.M1((1,1), [|tup|])
+    End Sub
+End Class"
+            Dim fix =
+"
+Friend Class C1
+    Private Sub M1(t1 As (Integer, Integer), tup As (Integer, String))
+        Dim tup=(1, ""1"")
+        Me.M1((1,1), tup)
+    End Sub
+End Class"
+            Await TestInRegularAndScriptAsync(code, fix)
+        End Function
+
+        <WorkItem(21446, "https://github.com/dotnet/roslyn/issues/21446")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)>
+        Public Async Function TestInvocationTuple3() As Task
+            ' error BC30057: Too many arguments to 'Private Sub M1(t1 As (Integer, Integer))'.
+            Dim code =
+"
+Friend Class C1
+    Private Sub M1(t1 As (Integer, Integer))
+        Dim tup=(i:=1, s:=""1"")
+        Me.M1((1,1), [|tup|])
+    End Sub
+End Class"
+            Dim fix =
+"
+Friend Class C1
+    Private Sub M1(t1 As (Integer, Integer), tup As (i As Integer, s As String))
+        Dim tup=(i:=1, s:=""1"")
+        Me.M1((1,1), tup)
+    End Sub
+End Class"
+            Await TestInRegularAndScriptAsync(code, fix)
+        End Function
+
+        <WorkItem(21446, "https://github.com/dotnet/roslyn/issues/21446")>
+        <Fact, Trait(Traits.Feature, Traits.Features.CodeActionsAddParameter)>
+        Public Async Function TestInvocation_OverloadResolution_Missing() As Task
+            ' error BC30311: Value of type 'Exception' cannot be converted to 'String'.
+            ' Overload resolution of VB gives only one Candidate M(String, String)
+            Dim code =
+"
+Public Class C
+    
+    Public Sub M(i1 As String, i2 As String)
+    End Sub
+    Public Sub M(ex As System.Exception)
+    End Sub
+
+    Public Sub Test()
+        M([|new System.Exception()|], 2)
+    End Sub
+End Class"
+            Await TestMissingAsync(code)
```

Comment:
The same test in C# offers a fix for the method with one parameter. The problem is that CandidateSymbols contains only one of the two overloads in VB (while in C# both methods are reported as candidates). I don't know if there is an easy way to also get the second (the fixable) overload. Maybe this is even a bug.

Reply:
Sounds like a potential bug to me.  @jcouv  can you advise?

Reply:
Definitely. Filed https://github.com/dotnet/roslyn/issues/25135 for follow-up on VB issue with `CandidateSymbols`.

Reply:
As suggested by @AlekseyTs in [comment](https://github.com/dotnet/roslyn/issues/25135#issuecomment-369682442) I will give methodgroup a try tomorrow. We have enough tests in place to make this a save change. @CyrusNajmabadi do you have any concerns?

Reply:
`MemberGroup` did the trick c225698.